OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -115,7 +118,7 @@ SimTK::ContactGeometry::TriangleMesh* ContactMesh::
     file.close();
     mesh.loadFile(filename);
     if (restoreDirectory) IO::chDir(savedCwd);
-        
+    _decorativeGeometry.reset(new SimTK::DecorativeMesh(mesh));
     return new SimTK::ContactGeometry::TriangleMesh(mesh);
```

Comment:
I'm not sure I understand; it seems like changing the return type of this function would still satisfy the requirement that the DecorativeMesh is only created if the ContactGeometry is created (with which I completely agree). We can chat in person.